lint.sh

```diff
@@ -1,26 +0,0 @@
-#!/bin/bash
```

Comment:
this lint file is currently useless. It was a copy-pasta from that in web sdk. Let's kill it for now and make a new one when we've got time in the future.